packages/analytics/amplify_analytics_pinpoint/lib/src/analytics_plugin_impl.dart

```diff
@@ -25,16 +27,25 @@ import 'package:meta/meta.dart';
 class AmplifyAnalyticsPinpoint extends AmplifyAnalyticsPinpointDart {
   /// {@macro amplify_analytics_pinpoint.analytics_plugin_impl}
   AmplifyAnalyticsPinpoint({
-    @visibleForTesting super.endpointInfoStore,
+    @visibleForTesting SecureStorageInterface? endpointInfoStore,
     @visibleForTesting CachedEventsPathProvider? pathProvider,
     @visibleForTesting AppLifecycleProvider? appLifecycleProvider,
     @visibleForTesting DeviceContextInfoProvider? deviceContextInfoProvider,
+    @visibleForTesting LegacyNativeDataProvider? legacyNativeDataProvider,
   }) : super(
+          keyValueStore: keyValueStore ??
```

Comment:
Should this be `endpointInfoStore`? It seems like this was renamed in some places and not in others.

Reply:
Thanks resolved.  I did a rebase and appear to have missed a commit somehow ...